bundles/core/org.eclipse.smarthome.core.thing/src/main/java/org/eclipse/smarthome/core/thing/type/ThingType.java

```diff
@@ -39,40 +37,35 @@
 
     private final List<ChannelGroupDefinition> channelGroupDefinitions;
     private final List<ChannelDefinition> channelDefinitions;
+    private final List<String> extensibleChannelTypeIds;
     private final List<String> supportedBridgeTypeUIDs;
-    private final Map<@NonNull String, String> properties;
+    private final @NonNull Map<@NonNull String, String> properties;
```

Comment:
As @maggu2810 already [suggested](https://github.com/eclipse/smarthome/pull/4301#pullrequestreview-63596110) we should use null annotations. According to [our guidelines](https://github.com/eclipse/smarthome/pull/4275/files#diff-24ba7e763168d1631a43ebdef19bcd5dR27) we should use `@NonNullByDefault` on the class level and remove it here.